examples/agentInteractionsDemo/app.js

```diff
@@ -207,9 +209,22 @@ new Vue({
                                 .then((conv) => {
                                     let newConv = conv;
                                     newConv.customer = conv.participants.find((part) => {
-                                        return part.purpose === "customer";
-                                    });
-                                    this.conversationsData.conversations.push(newConv);
+                                        return part.purpose === "customer" || part.purpose === "external";
```

Comment:
Since you're using this in two places, would it make sense to create a util method, getCustomerParticipant(conversation) that encapsulates this logic?  If it returns null, you can still handle the errors here where the call is made.

This will also let you improve this logic a bit.  For instance, you might want to return the last customer or external participant vs the first.  There are some other nuanced cases for detecting the customer, but, it's probably overkill for this demo.